pages/samvera/developer_community/git_structure/core_components.md

```diff
@@ -28,7 +28,7 @@ that falls into disuse should be a candidate for [deprecation](deprecation.html)
 
 ## Maintenance
 
-The Component Interest Group (CIG) is in the process of creating a framework for
+The Component Maintenance Interest Group (CIG) is in the process of creating a framework for
```

Comment:
Minor thing, but how about removing the acronym?  I don't see it used anywhere else in the document and with this name change doesn't match.
```suggestion
The Component Maintenance Interest Group is in the process of creating a framework for
```